ci-operator/config/openshift/origin/openshift-origin-master.yaml

```diff
@@ -43,10 +38,6 @@ images:
 - context_dir: images/os/
   from: base
   inputs:
-    base-machine-with-rpms:
```

Comment:
So I think specifically the `machine-os-content` build described here still requires the fedora image for ... something ... with the multistage build.